content/docs/installation-try-react.md

```diff
@@ -0,0 +1,21 @@
+---
+id: installation-try-react
+title: Try React
+permalink: docs/installation-try-react.html
+redirect_from:
+  - "docs/installation.html"
+  - "download.html"
+  - "downloads.html"
+  - "docs/tooling-integration.html"
+  - "docs/package-management.html"
+  - "docs/language-tooling.html"
+  - "docs/environments.html"
+---
+
+React is flexible and can be used in a variety of projects. You can create new apps with it, but you can also gradually introduce it into an existing codebase without doing a rewrite.
```

Comment:
I'm not sure this lead paragraph (and the below header) make sense anymore, given what this page has been remade into.

Maybe we should rename the page title to something like "Try it on CodePen" and leave out this paragraph entirely?

Reply:
Sure, although, how about adding more than Codepen? The Codesandbox template for React is pretty mature and quite popular, and we could link out to other content like the editable examples on the home page.

Reply:
Sure. That'd be totally fine. CodeSandbox comes to mind as another excellent solution.